env/CMakeLists.txt

```diff
@@ -47,17 +47,19 @@ ExternalProject_Add(
     -DCMAKE_CXX_COMPILER=clang++
     -DLLVM_ENABLE_PROJECTS=mlir
     -DLLVM_INSTALL_UTILS=ON
+    -DLLVM_TARGETS_TO_BUILD=all
```

Comment:
all changes to this file will be undone eventually--this line specifically doesn't do anything, since "all" is default value